onnx/checker.cc

```diff
@@ -1022,8 +1041,9 @@ std::set<std::string> experimental_ops = {
     "Scale",
     "ScaledTanh"};
 
-bool check_is_experimental_op(std::string node_op_type) {
-  return (experimental_ops.count(node_op_type)) ? true : false;
+bool check_is_experimental_op(const NodeProto& node) {
+  return (node.domain() == ONNX_DOMAIN || node.domain() == "ai.onnx") && experimental_ops.count(node.op_type()) ? true
```

Comment:
Looks good, thanks!

Another nit (about the old code): I think the ` ? true : false` at the end is unnecessary and can be omitted?

Reply:
Good catch.. Will update it soon. Thanks!